docs/source/deployment-guide/deployment-guide-for-minimax-m3-on-trtllm.md

```
@@ -0,0 +1,364 @@
+          --trust_remote_code \
+          --reasoning_parser minimax_m3 \
+          --tool_parser minimax_m3 \
+          --chat_template $MODEL/chat_template.jinja \
+          --host 0.0.0.0 \
+          --extra_llm_api_options /workspace/extra_llm_api.yaml"
+```
+
+> [!NOTE]
+> Adjust `-N`, `--ntasks`, `--ntasks-per-node`, and `--gres=gpu:` to match your cluster's GPUs-per-node. The total number of tasks (ranks) must equal `--tp_size` (`8`). Add the partition / account / node-list flags (`-p`, `-A`, `-w`) required by your Slurm setup, and ensure `/models` and `/workspace` resolve to the same shared paths on both nodes.
+> To run the **TEP** configuration instead, remove the `--enable_attention_dp` flag.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)